.prettierrc

```
@@ -0,0 +1,14 @@
+{
+  "printWidth": 80,
```

```suggestion
  "printWidth": 120,
```
May we fast forward to the year 2024, where we are no longer limited to 1990 era 80-column VGA monitors? 😉 Such narrow column wraps often hamper reading by causing premature line wrapping, lines which would be easier to read if they weren't raggedly split up. ORT uses 120. DML generally limits itself to that (unless it would worsen readability for that line, particularly for tables). Granted, not everyone has a 4k monitor, but mine can display up to 4 open files side by side at 120 columns.